### PR TITLE
Lpd 42555 details entries

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/LPD-42555

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/constants/CommerceProductFDSNames.java [hunk 1]
```diff
@@ -27,6 +27,9 @@ public class CommerceProductFDSNames {
 	public static final String PRODUCT_CONFIGURATION_LISTS =
 		CPPortletKeys.CP_CONFIGURATION_LISTS + "-productConfigurationLists";
 
+	public static final String PRODUCT_CONFIGURATIONS =
```
Comment (smottal) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/constants/CommerceProductFDSNames.java:30]: why is not PRODUCT_CONFIGURATION_ENTRIES? I think we are using the name productConfiguration (for the entries) only in the API

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by smottal** [comment]
@andrea-ale-sbarra , are more questions / doubts that changes needed

File: modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/configuration_list/entries.jsp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<%
+CPConfigurationListDisplayContext cpConfigurationListDisplayContext = (CPConfigurationListDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+long cpConfigurationListId = cpConfigurationListDisplayContext.getCPConfigurationListId();
+%>
+
+<commerce-ui:panel
+	bodyClasses="p-0"
+	elementClasses="mt-4"
+	title='<%= LanguageUtil.get(request, "products") %>'
```
[the remark this answers is not in the document]
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/configuration_list/entries.jsp:19]: yes it is like so in the mockups

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/security/permission/resource/CPConfigurationEntryModelResourcePermission.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+			PermissionChecker permissionChecker, long cpConfigurationEntryId,
+			String actionId)
+		throws PortalException {
+
+		CPConfigurationEntry cpConfigurationEntry =
+			cpConfigurationEntryLocalService.getCPConfigurationEntry(
+				cpConfigurationEntryId);
+
+		CommerceCatalog commerceCatalog =
+			commerceCatalogLocalService.fetchCommerceCatalogByGroupId(
```
[the remark this answers is not in the document]
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/security/permission/resource/CPConfigurationEntryModelResourcePermission.java:88]: technically yes, but it should be never null in here I just didn't want to throw unnecessary exceptions

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**